addons/help/XEH_postInit.sqf

```diff
@@ -37,7 +37,7 @@ if (!hasInterface) exitWith {};
     // delete last line breaks
     _keys = _keys select [0, count _keys - 10];
 
-    _unit createDiaryRecord [QGVAR(docs), [localize "STR_CBA_Help_Keys", _keys]];
-    _unit createDiaryRecord [QGVAR(docs), [localize "STR_CBA_Credits", call (uiNamespace getVariable QGVAR(credits))]];
-    _unit createDiaryRecord [QGVAR(docs), [localize "STR_CBA_Addons", call (uiNamespace getVariable QGVAR(mods))]];
+    _unit createDiaryRecord [QGVAR(docs), [localize "STR_CBA_Help_Keys", format["<font size=20>%1</font><br/>%2", localize "STR_CBA_Help_Keys", _keys], taskNull, "", false]];
```

Comment:
```suggestion
    _unit createDiaryRecord [QGVAR(docs), [localize "STR_CBA_Help_Keys", format ["<font size=20>%1</font><br/>%2", localize "STR_CBA_Help_Keys", _keys], taskNull, "", false]];
```

Reply:
one of the `]` was also wrong at the end. I missed it and the error in testing.